### DISCUSSION TITLE
Issue with accessing ArchiveBox via localhost:8000 when running on port 8000 inside a VM

### DISCUSSION THREAD

**Discussion by hypn0toad** [ArchiveBox/ArchiveBox#791] [Q&A] [answered]
Hi I'm pretty new to archivebox and docker but seem to be having an issue. I'll start docker and try to do an init, which works. Then I'll try to run the server via the commands at the top of the github page.

The server says that it starts running, but when I try to goto localhost:8000, I get unable to connect in my web browser. In the docker console things look good, and i see messages like ""GET /admin/login/ HTTP/1.1" 200 11144". The docker getting started demo works fine.

The funny thing is this was working just fine earlier in the week (I had setup an initial archive). Then I came back later and windows patched and it didn't work. 

I thought it might be a docker/windows issue so I tried on another computer, and in a new VM running ubuntu and have the same problem with each one. Am I going crazy? I tried it on

- Windows 10 + WSL + Docker (comp 1 running v3.5.2)
- Windows 10 + WSL + Docker (Comp 2 running a downgraded v3.4.0)
- VM running Ubuntu 20.04 LTS

I'm trying to create a new archivebox from scratch. And am just running init and then the run. Either via docker compose or via regular docker messages. I've tried varying the port with no success. And have tried on multiple browsers.

For the VM I just ran
```bash
mkdir ~/archivebox && cd ~/archivebox
docker run -v $PWD:/data -it archivebox/archivebox init --setup
docker run -v $PWD:/data -p 8000:8000 archivebox/archivebox
```

Is anyone else seeing issues? I figured I'd post on the forum not as an issue since I figure the problem is on my end...

Thank you!

**Comment by pirate** [comment, marked as answer]
Your VM likely has a different IP than your host machine, or is NATed/behind a firewall, have you tried visiting `http://<vm ip here>:8000` instead? Also check your WSL/VM settings to make sure ports opened inside are exposed to the host.

**Reply by hypn0toad** [reply]
I was trying to go to the website within the VM so I didn't think I would have IP issues like that but i can check.

**Reply by pirate** [reply]
Ah yeah that shouldn't be an issue then. Can you screenshot the error page from the browser and the terminal where you're running the commands to start the archivebox server. It's helpful to see the exact error messages and log output.

**Reply by hypn0toad** [reply]
Okay i just did it again and didn't have a problem everything worked. The only thing that is different is I used a different directory and skipped setting my email address. I'll mess around with this some more later today to see if I can narrow down what i was doing wrong but for the moment everything seems okay. Sorry!
